app/helpers/form_helper.rb

```diff
@@ -192,6 +192,18 @@ def autocomplete_f(f, attr, options = {})
     end
   end
 
+  def byte_size_f(f, attr, options = {})
+    base_id = ActionView::Helpers::Tags::TextField.new(f.object_name, attr, nil, options).send :add_default_name_and_id, {}
```

Comment:
This is a private function that may break in the future, why not just push the hidden field into one of the help blocks with no id and find it like we do for the warning message?